### PR TITLE
Feature/email confirmation

### PR DESCRIPTION
**Pull request by MahdiZareie**

### CODE REVIEW

**Comment by MahdiZareie** [comment]
https://github.com/fandoghpaas/fandogh-cli/issues/50
Please do not merge this PR yet @psycho-ir

**Comment by MahdiZareie** [comment]
I'm done with this branch, please review PR so we can merge and continue.

@psycho-ir

File: server/fandogh/user/views.py [hunk 1]
```diff
@@ -62,13 +64,31 @@ def post(self, request):
...
                         return GeneralResponse("User has been registered successfully", status=status.HTTP_201_CREATED)
             except Exception as e:
                 print(e)
                 return ErrorResponse("A user with current email or namespace exists", status=status.HTTP_400_BAD_REQUEST)
         else:
             return ErrorResponse(serialized.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+class ActivationView(APIView):
+    def patch(self, request, activation_code):
```
Comment (psycho-ir) [inline server/fandogh/user/views.py]: why is it patch?

Comment (MahdiZareie) [reply server/fandogh/user/views.py]: because it's suppose to change the state of a ActivationCode object.

**Pull request merged**